config/sidebar.yml

```diff
@@ -549,6 +549,20 @@ articles:
             url: "/multifactor-authentication/developer/step-up-authentication/step-up-for-apis"
             hidden: true
 
+      - title: "API"
```

Comment:
Lets leave the addition in the sidebar for when the feature goes live for all customers. We should have the PR with the changeset ready, standing by for Product green light.